src/main/java/reposense/model/RepoConfiguration.java

```diff
@@ -219,9 +219,17 @@ public String getBranch() {
     }
 
     public void setBranch(String branch) {
+        updateDisplayName(branch);
         this.branch = branch;
     }
 
+    /**
+     * Update display name
+     */
+    public void updateDisplayName(String branch) {
```

Comment:
The current header comment is exactly what the method name is saying, which makes it redundant. Can just remove it.

Reply:
has been removed